rts/Game/GameSetup.cpp

```diff
@@ -582,6 +584,7 @@ bool CGameSetup::Init(const std::string& buf)
 
 	file.GetTDef(mapSeed, unsigned(0), "GAME\\MapSeed");
```

Comment:
Looks like "map seed" exists which makes "fixed seed" not obviously RNG related. Perhaps call it "FixedRNGSeed" or something? Otherwise looks good though.

Reply:
FixedRNGSeed is much more clear. Changing that 🙌

Reply:
done 👍